rpm/voicecall-qt5.spec

```diff
@@ -123,3 +131,7 @@ fi
 %defattr(-,root,root,-)
 %{_libdir}/voicecall/plugins/libvoicecall-ofono-plugin.so
 
+%files plugin-voicecall-filter
+%defattr(-,root,root,-)
```

Comment:
Exists already on other packages, but no need to have these defattrs anymore, it's the default.